docusaurus.config.ts

```diff
@@ -73,38 +65,37 @@ const config: Config = {
       },
       items: [
         { to: '/', label: 'Home', position: 'left', activeBaseRegex: `///` },
-        { to: '/docs',  position: 'left', label: 'Documentation' },
-        { to: '/changelog',  position: 'left', label: 'Changelog' },
-        { href: 'https://meroxa.com/blog', position: 'left', label: 'Blog' },
+        { to: '/docs', position: 'left', label: 'Documentation' },
+        { to: '/changelog', position: 'left', label: 'Changelog' },
         { to: '/api', label: 'HTTP API', position: 'left' },
-        { href: 'https://github.com/ConduitIO/conduit/discussions', label: 'GitHub Discussions', position: 'right' },
-        { href: 'https://discord.meroxa.com', label: 'Discord Community', position: 'right' },
+        { href: 'https://meroxa.io', label: 'Conduit Platform', position: 'right', class: 'navbar__link navbar__item conduit-platform' },
+        { href: 'https://meroxa.com/blog', position: 'right', label: 'Blog' },
```

Comment:
I'm thinking, this blog is about all things Meroxa. Someone visiting our site and clicking "Blog" will likely expect and would like to see Conduit-related articles. I'm wondering if it makes sense to link to the Conduit tag instead?

Reply:
We can use this https://meroxa.com/blog/tag/conduit/, no?